content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -154,7 +154,7 @@ In addition, `node.kubernetes.io/unschedulable:NoSchedule` toleration is added
 automatically to DaemonSet Pods. The default scheduler ignores
 `unschedulable` Nodes when scheduling DaemonSet Pods.
 
-When this feature is enabled, if DaemonSet uses the host network, you must also add the
+If DaemonSet uses the host network, you must also add the
```

Comment:
When https://github.com/kubernetes/kubernetes/pull/64954 merged, this's not necessary.

Reply:
@k82cn Can you place a note in kubernetes/kubernetes#64954 to remind someone to update this page when it is merged?

Reply:
@zparnold no worries, I have an eye on that PR.

Reply:
Resolved.